ajax/libs/multiple-select/package.json

```diff
@@ -28,7 +28,7 @@
       {
         "basePath": "",
         "files": [
-          "multiple.select.*"
+          "multiple-select*.+(js|css|map)"
```

Comment:
We also need .png file. So let us modify this as `multiple-select*.+(js|css|map|png)`.